docs/conf/portland/2019/news/events-activities.rst

```diff
@@ -0,0 +1,64 @@
+
+Diversity scholarship tickets are **free**, and we hope to welcome more people from all walks of life to join our community.
+Please share this campaign with your social networks and encourage folks to apply if they are interested to join us!
+
+Shirts on sale
+--------------
+
+Following last year's pilot, we are happy to announce that the `Write the Docs Portland 2019 Pop-Up Shop <https://teespring.com/wtd-portland-2019>`_ is now open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want!
+
+The campaign will run in 3-day batches until **May 30th**. We recommend to order your shirt at least 2-3 weeks in advance, to allow for timely delivery of your shirts ahead of the conference.
+
+Similarly, we will be continuing with our bag swag this year, which you can pick up at the conference.
+
+Events and activities
+---------------------
+
+Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge.
+
+* **Hike**. If you're in town early, join us on Saturday for our traditional `hike <https://www.writethedocs.org/conf/portland/2019/hike/>`_ is a great chance to get to know your fellow documentarians in a smaller setting. This activity is free with your conference ticket, but space is limited! We will publish a separate registration soon.
+* **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/portland/2019/writing-day/>`_ will be held on Sunday from *9am-5pm*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *9:30am*. Please email us if you want to submit a project for the Writing Day! We will publish a list of submitted projects closer to the day.
+* **Welcome reception.** The meet-and-greet event will be held on Sunday from *5pm-8pm*. You can pick up your badge and enjoy a couple of drinks on us. This is a great chance to meet your fellow documentarians before the conference starts.
+* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/2019/lightning-talks/>`_ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
+* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/2019/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a very specific thing? This is the place. We'll have a sign-up board so you know what's happening, and supplies to help you brainstorm.
```

Comment:
```suggestion
* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/2019/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place. We'll have a sign-up board so you know what's happening, and supplies to help you brainstorm.
```
I'd drop the "very", it feels to me like there is a minimum amount of specificity needed for an unconference session, but some are quite broad.